ak_vendor/enums.py

```diff
@@ -455,3 +455,15 @@ class Meta:
         NONE = [0, "None"]
         TOTP = [1, "TOTP"]
         HOTP = [2, "HOTP"]
+
+
+@choices
+class FileTypeEnum:
+    """
+    The file types.
+    """
+
+    class Meta:
+        ANDROID_APK = [0, "Android APK"]
+        ANDROID_AAB = [1, "Android AAB"]
+        IOS_IPA = [2, "IOS IPA"]
```

Comment:
```suggestion
        ANDROID_APK = [0, "APK"]
        ANDROID_AAB = [1, "AAB"]
        IOS_IPA = [2, "IPA"]
```
Because the abbreviation expansion itself contains the platform name (APK - Android application PacKage, AAB - Android App Bundle, IPA - iOS Appstore Package).. Let the key be prefixed for better readability